tests/e2e/blocks/audio-video.spec.js

```diff
@@ -0,0 +1,71 @@
+import {test, expect} from '../tools/lib/test-utils.js';
+import {publishPostAndVisit, createPostWithFeaturedImage} from '../tools/lib/post.js';
+import {searchAndInsertBlock} from '../tools/lib/editor.js';
+
+const YOUTUBE_TEST = 'https://www.youtube.com/watch?v=-CwkccAgKrs';
+const VIMEO_TEST = 'https://vimeo.com/100195272';
+const MP4_TEST = 'https://www.greenpeace.org/static/planet4-assets/tests/edge_of_the_wrold.mp4';
+
+const SOUNDCLOUD_TEST = 'https://soundcloud.com/greenpeaceuk-1/04-and-we-will-defend-requiem';
+const MP3_TEST = 'https://www.greenpeace.org/static/planet4-assets/tests/wochenserie_greenpeace.mp3';
+
+/**
+ * Add a Video or Audio block to a page with a specific link.
+ *
+ * @param {Object} page - The page object for interacting with the browser.
+ * @param {string} mediaLink - The video file link (can be YouTube, Vimeo, mp4).
+ * @param {string} mediaType - The type of media added (Audio or Video).
+ */
+const addVideoOrAudioBlock = async ({page}, mediaLink, mediaType) => {
+  await searchAndInsertBlock({page}, mediaType);
+  await page.getByRole('button', {name: 'Insert from URL'}).click();
+  await page.getByPlaceholder('Paste or type URL').fill(mediaLink);
+  await page.keyboard.press('Enter');
+};
+
+test.useAdminLoggedIn();
+
+test('check the Audio and Video blocks', async ({page, admin, editor}) => {
+  // Make sure the "Lazy Youtube player" setting is enabled.
+  await page.goto('./wp-admin/admin.php?page=planet4_settings_features');
+  const lazyYoutubePlayerSetting = page.locator('#lazy_youtube_player');
+  const alreadyEnabled = await lazyYoutubePlayerSetting.isChecked();
+  if (!alreadyEnabled) {
+    await lazyYoutubePlayerSetting.check();
+    await page.locator('input[type="submit"]').click();
+  }
+
+  // Create a post for the test.
+  await createPostWithFeaturedImage({admin, editor}, {title: 'Test Audio and Video blocks'});
```

Comment:
```suggestion
  await createPostWithFeaturedImage({page, admin, editor}, {title: 'Test Audio and Video blocks'});
```

Reply:
Why this change? The [function definition](https://github.com/greenpeace/planet4-master-theme/blob/b2625c0e0731ac92f30286f17cde0b3ee7d4531e/tests/e2e/tools/lib/post.js#L56) doesn't have the `page` param

Reply:
Well, I'm testing it through [this branch](https://github.com/greenpeace/planet4-master-theme/pull/2419/files#diff-6d6017df1507cc170b060aa14a2a7512b35d9db51ddc4bab83ae8de6ed05008cR59) 🙃